clients/drcachesim/simulator/caching_device_stats.cpp

```diff
@@ -163,10 +164,11 @@ caching_device_stats_t::dump_miss(const memref_t &memref)
         pc = memref.data.pc;
     }
     addr = memref.data.addr;
+    pid = memref.data.pid;
 #ifdef HAS_ZLIB
-    gzprintf(file_, "0x%zx,0x%zx\n", pc, addr);
+    gzprintf(file_, "%d,0x%zx,0x%zx\n", pid, pc, addr);
```

Comment:
The documentation is now wrong: please update the docs ("Each miss is written in text format as a <program counter, address> pair.")

Reply:
I have already update the options.cpp file by changing the original statement with ("Each miss is written in text format as a <process id, program counter, address> tuple.")